src/components/control/multi-autocompleter.jsx

```diff
@@ -13,10 +13,17 @@ const MultiAutocompleter = forwardRef(function MultiAutocompleter(props, ref) {
   const { execute: fetchSelectedChoices } = useFetch();
   const [loading, setLoading] = useState(true);
   const [options, setOptions] = useState([]);
-  const [value, setValue] = useState(props.value);
+  const mounted = useRef(false);
+
+  function arrayWrap(v) {
+    return Array.isArray(v) ? v : [v];
+  }
+
+  const [value, setValue] = useState(arrayWrap(props.value));
+  const [valueForSelect, setValueForSelect] = useState(value);
```

Comment:
From my understanding between `value` and `valueForSelect`, this initial state might not be entirely correct.

`value` is always an array of objects or strings. `valueForSelect` is always an array of objects 
 -- which is populated after fetching the objects from the API. 

Does it make more sense to ensure `valueForSelect` is a list of objects in the initial state? Otherwise I think we're violating the prop-types for `value` on the MultiSelect component. Something like: 

```suggestion
  const [valueForSelect, setValueForSelect] = useState(value.map(v => typeof v === 'string' ? { id, label: '' } : v));
```